src/tests/python/sssd/testlib/common/libkrb5.py

```diff
@@ -198,20 +198,15 @@ def destroy_krb5server(self):
             :Exception: subprocess.CalledProcessError
         """
         # stop the Kerberos server
-        try:
-            self.multihost.run_command(['systemctl', 'stop', 'krb5kdc'])
-        except subprocess.CalledProcessError:
-            raise
-        else:
-            self.multihost.log.info("stopped krb5kdc service")
-
-        # stop kadmin service
-        try:
-            self.multihost.run_command(['systemctl', 'stop', 'krb5kdc'])
-        except subprocess.CalledProcessError:
-            raise
-        else:
-            self.multihost.log.info("stopped kadmin service")
+        services = ['krb5kdc', 'kadmin']
```

Comment:
I think it would be more pythonic to use a tuple instead of a list, since the data is not mutable. Also, you could equally well just use the tuple literal in the for loop later, e.g.:
```
for service in ('krb5kdc, 'kadmin')
```